src/trace/mod.rs

```diff
@@ -253,7 +254,7 @@ where
     /// Values associated with keys.
     type Val;
     /// Timestamps associated with updates
-    type Time;
+    type Time: Lattice+Timestamp;
```

Comment:
I'm not fundamentally opposed, but is there a reason this type needs a constraint but the others do not?

Reply:
Removed, stray.